File: WEBAPI/Models/DTOs/Shipping/Econt/AddressDTO.cs
namespace Models.DTOs.Shipping.Econt
{
    public class AddressDTO
    {
        public int? Id { get; set; }
        public CityDTO? City { get; set; }
        public string? Street { get; set; }
        public string? Num { get; set; }
        public string? Other { get; set; }
        public LocationDTO? Location { get; set; }
        public string? FullAddress { get; set; }
        public string? FullAddressEn { get; set; }
        public string? Quarter { get; set; }
        public string? Zip { get; set; }
        public string? Hezid { get; set; }
    }
}

File: PROJECT/FRONTEND/Common/DTOs/AddressDTO.ts


export class AddressDTO { 
    public id: number | null;
    public city: CityDTO;
    public street: string | null;
    public num: string | null;
    public other: string | null;
    public location: LocationDTO;
    public fullAddress: string | null;
    public fullAddressEn: string | null;
    public quarter: string | null;
    public zip: string | null;
    public hezid: string | null;
}
